go/mysqlconn/conn.go

```diff
@@ -515,6 +515,10 @@ func (c *Conn) RemoteAddr() net.Addr {
 	return c.conn.RemoteAddr()
 }
 
+func (c *Conn) Ident() string {
+	return fmt.Sprintf("client %v (%s)", c.ConnectionID, c.RemoteAddr().String())
```

Comment:
i think we're trying to find if any particular client connects slower than others, it might help the logstash search if the identity string formats like, `client=(%s) conn=%v`, which is simpler to verify a suspect client ip